test/react-native/features/fixtures/app/react_native_navigation_js/scenarios/ReactNavigationBreadcrumbsEnabledScenario.js

```diff
@@ -0,0 +1,10 @@
+import Scenario from './Scenario'
+
+export class ReactNavigationBreadcrumbsEnabledScenario extends Scenario {
+  constructor(configuration, extraData, jsConfig) {
+    super()
+  }
+
+  run() {
+  }
+}
```

Comment:
I think can literally be empty as the required methods are all defined in `Scenario`

```suggestion
export class ReactNavigationBreadcrumbsEnabledScenario extends Scenario {}
```